backend/de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/payment/paymentallocation/service/AllocationLineCandidate.java

```diff
@@ -114,4 +119,14 @@ private AllocationLineCandidate(
 		this.paymentOverUnderAmt = paymentOverUnderAmt != null ? paymentOverUnderAmt : Money.zero(amounts.getCurrencyId());
 		this.invoiceProcessingFeeCalculation = invoiceProcessingFeeCalculation;
 	}
+
+	public static class AllocationLineCandidateBuilder
+	{
+		public AllocationLineCandidateBuilder payableDocument(@NonNull final PayableDocument payableDocument)
+		{
+			// payableDocumentRef(payableDocument.getReference()); TODO remove
```

Comment:
why not setting the reference?
and anyways, if reference is not set, what's the purpose of this method?

Reply:
misread it. But I brought it back soon after I got there with the debugger.